cisc/commands.go

```diff
@@ -359,5 +359,82 @@ func getCommands() cli.Commands {
 				},
 			},
 		},
+
+		{
+			Name:    "cert",
+			Aliases: []string{"c"},
+			Usage:   "create and use links with admin privileges",
+			Subcommands: cli.Commands{
+				{
+					Name:      "request",
+					Aliases:   []string{"q"},
+					Usage:     "request a certificate to letsencrypt and store it to the skipchain",
+					ArgsUsage: "domain-name cert-dir www-dir",
+					Action:    certRequest,
+				},
+				{
+					Name:      "list",
+					Aliases:   []string{"l"},
+					Usage:     "List the certificate",
+					ArgsUsage: "",
+					Action:    certList,
+					Flags: []cli.Flag{
+						cli.BoolFlag{
+							Name:  "verbose, v",
+							Usage: "Display the fullchain certificate",
+						},
+						cli.BoolFlag{
+							Name:  "public, p",
+							Usage: "Display the public certificate",
+						},
+						cli.BoolFlag{
+							Name:  "chain, c",
+							Usage: "Display the chain certificate",
+						},
+					},
+				},
+				{
+					Name:      "verify",
+					Aliases:   []string{"v"},
+					Usage:     "verify the certificate against the root certificate",
+					ArgsUsage: "key of the certificate",
```

Comment:
`ArgsUsage: "cert_key [skipchain-ID]"`
The `ArgsUsage` should be one word per argument, so the user knows that it's one argument.